plugins/modules/system/sudoers.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+# Copyright: (c) 2019, Jon Ellis (@JonEllis0) <ellis.jp@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: sudoers
+short_description: Manage Linux sudoers files
+description:
+    - This module allows for the manipulation of Linux sudoers files
+author:
+    - "Jon Ellis (@JonEllis0) <ellis.jp@gmail.com>"
+options:
+    command:
+        description:
+            - The command allowed by the sudoers rule
+            - Multiple can be added by passing a list of commands
+        type: list
+        elements: str
+    group:
+        required: false
+        description:
+            - Name of the group for the sudoers rule
+            - (cannot be used in conjunction with user)
+        type: str
+    name:
+        required: true
+        description:
+            - Name of the sudoers rule
+        type: str
+    nopassword:
+        required: false
+        description:
+            - Whether a password will be required to run the sudo'd command
+        default: true
+        type: bool
+    state:
+        required: false
+        default: "present"
+        choices: [ present, absent ]
+        description:
+            - Whether the rule should exist or not
+        type: str
+    user:
+        required: false
+        description:
+            - Name of the user for the sudoers rule
+            - (cannot be used in conjunction with group)
+        type: str
+'''
+
+EXAMPLES = '''
+- name: Allow the backup user to sudo /usr/local/bin/backup
+  sudoers:
+    name: allow-backup
+    state: present
+    user: backup
+    command: /usr/local/bin/backup
+
+- name: Allow the monitoring group to run sudo /usr/local/bin/gather-app-metrics without requiring a password
+  sudoers:
+    name: monitor-app
+    group: monitoring
+    command: /usr/local/bin/gather-app-metrics
+
+- name: Allow the alice user to run sudo /bin/systemctl restart my-service or sudo /bin/systemctl reload my-service, but a password is required
+  sudoers:
+    name: alice-service
+    user: alice
+    command:
+      - /bin/systemctl restart my-service
+      - /bin/systemctl reload my-service
+    nopassword: false
+
+- name: Revoke the previous sudo grants given to the alice user
+  sudoers:
+      name: alice-service
+      state: absent
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+import os
```

Comment:
nit picking: by convention, python standard modules should come first. Not that it makes much of a difference in the result here but hey, nit-pickers of the world, unite!

Reply:
Nit pick away :)